tracim/tracim
# Feat/5655  remove raw_content from events

related to #5655 

## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**


Please, test:
- Edit/Delete/Restore a note
- TLMs when another user edit/delete/restore a note
- Create/Edit/Delete a comment
- TLMs when another user create/edit/delete a comment
- Create/Edit/Delete/Restore a news
- TLMs when another user create/edit/delete/restore a news
- See in your console if `content.*.*` TLMs do not have the `raw_content` field
